internal/scheduler/schedulerobjects/schedulerobjects.proto

```diff
@@ -21,6 +21,8 @@ message Executor {
     google.protobuf.Timestamp lastUpdateTime = 5 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
     // Jobs that are owned by the cluster but are not assigned to any node.
     repeated string unassigned_job_runs = 9;
+    
+    int64 num_unacknowledged_jobs = 10;
```

Comment:
Why do we need this? The scheduler should already know which jobs haven't been acked.